internal/definitions/handler.go

```diff
@@ -80,16 +80,16 @@ type definitionHandler struct {
 	namespace  string
 	multiparty bool
 	database   database.Plugin
-	blockchain blockchain.Plugin
+	blockchain blockchain.Plugin   // optional
 	exchange   dataexchange.Plugin // optional
 	data       data.Manager
 	identity   identity.Manager
 	assets     assets.Manager
-	contracts  contracts.Manager
+	contracts  contracts.Manager // optional
```

Comment:
Since two more items became "optional" (and may now be nil), are there any places that need additional nil checks? Or they're on paths that are guaranteed not to be hit?